packages/astro/src/@types/astro.ts

```diff
@@ -1094,6 +1094,29 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		inlineStylesheets?: 'always' | 'auto' | 'never';
+		/**
+		 * @docs
+		 * @name build.concurrency
+		 * @type {boolean}
```

Comment:
```suggestion
		 * @type { number }
```
Is this number? (And I don't remember, does it go in curly braces?)